showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+The data table of all the samples was made available at the access restricted website for the Health Board employees and project participants to provide epidemiological and virological consultations. 
+
+The database included all metadata provided for the samples, sequencing outcomes and was interactive providing filtering and sorting among other customisation options. In addition data could be downloaded for further custom analysis. The closed website of the project dashboard included also different interactive plots that allowed the users to pick the variables to be plotted (e.g. Ct values against vaccination status, county against lineages), but also lineage overview where individual lineages could be included or dismissed, time frame to be selected. 
+
+In addition, for each sequenced plate mutation plots and Ct value plots were implemented to carry out visual inspection as part of the quality control of the sequencing outcomes. In all the dashboards, users could select if they wanted to include data only sequenced in Estonia, by ECDC or both. 
+
+### Public dashboards
+In addition to the access restricted dashboards, public data on the clades, percentage of all positive samples in Estonia been sequenced and clade distribution across counties, regions and age groups were made available and regularly updated at the [Estonian COVID19Dataportal](https://covid19dataportal.ee/genomics_transcriptomics/).  In order to illustrate the virus evolution a public [Nextstrain Auspiece](https://auspice.biit.cs.ut.ee/ncov/est) instance was set up. In regular time intervals, all samples were rerun to obtain the latest lineage annotations.
+
+## Data brokering to ENA 
+
+To upload data to [European Nucleotide Archive (ENA)](https://www.ebi.ac.uk/ena/browser/) we registered our research [projects at the ENA](https://www.ebi.ac.uk/ena/browser/text-search?query=korogeno-est) following the [guideline](https://ena-docs.readthedocs.io/en/latest/submit/study.html). At the beginning we created individual projects for a data batch but later joined many under one. The Data broker generated ENA metadata files using ENA virus pathogen reporting standard checklist ERC000033 filling out mandatory and recommended fields ([checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000033), [metadata template](https://github.com/ELIXIR-Belgium/ENA-metadata-templates)).  The preparatory work for data brokering to ENA was done by a dedicated Galaxy workflow. A typical workflow of extraction of Galaxy results to the metadata file and metadata ERC000033 file generation took about 1h. 
+
+Out of the three different raw sequencing data uploading ways, Estonia used the Galaxy-CLI instance which is a wrapper for the ENA upload tool. The wrapper is [available](https://github.com/usegalaxy-eu/ena-upload-cli) along with the [dedicated guide](https://rdm.elixir-belgium.org/covid-19/sarscov2_submission.html) that was [published](https://doi.org/10.1093/bioinformatics/btab421) in 2021.
```

Comment:
```suggestion
Estonia used the Galaxy [ENA Upload container](https://github.com/ELIXIR-Belgium/ena-upload-container) from the [ENA Data Submission Toolbox](https://www.elixir-belgium.org/services/ena-data-submission-toolbox) to process and submit raw genome sequencing reads to ENA. This container allows for the deployment of a Galaxy instance containing a wrapped Command-line-interface ENA Upload Tool, providing a graphical user interface and reproducible data submission pipeline. Documentation is available in [dedicated pages](https://rdm.elixir-belgium.org/covid-19/sarscov2_submission) in the RDM Guide, and a standalone [publication](https://doi.org/10.1093/bioinformatics/btab421).
```

Attempted a re-write to make the name of the tools a bit clearer.

Reply:
ok